Application/Telegram/Handlers/UpdateCallbackQuery.cs

```diff
@@ -33,13 +37,21 @@ public static async Task Handle(Update update, TelegramBotClient telegraBotClien
                     case "New Booking":
                         UserStateStorage.UserStateUpdate(update.CallbackQuery.From.Id, UserState.NewBookingIsSelected);
                         await new ProvideButtons(telegraBotClient).SendAsync(
-                            update.CallbackQuery, new List<string>() { "Search by location", "Next", "BACKStartingProcess" }, 2);
+                             update.CallbackQuery
+                            ,new List<string>() { "Next", "BACK" }
```

Comment:
it seems while resolving conflict you deleted "Search by location" button. 
it was like that 

                update.CallbackQuery, new List<string>() { "New Booking", "My Bookings", "New Vacation", "My Vacations", "BACKProcessNotStarted" }, 2);

Reply:
done